src/Microsoft.Data/StringColumn.BinaryOperations.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Data
+{
+    public partial class StringColumn : BaseColumn
+    {
+        public override BaseColumn Add(BaseColumn column)
+        {
+            // TODO: Using indexing is VERY inefficient here. Each indexer call will find the "right" buffer and then return the value
```

Comment:
Can we log issues for TODOs? What is the plan to address them?

Reply:
Filed [this](https://github.com/dotnet/corefxlab/issues/2681) to address this